cinq_collector_aws/views/elbs.py

```diff
@@ -0,0 +1,89 @@
+'''views for cinq_collector_elb'''
+
+from cloud_inquisitor.constants import ROLE_USER, HTTP
+from cloud_inquisitor.plugins import BaseView
+from cloud_inquisitor.utils import MenuItem
+from cloud_inquisitor.wrappers import check_auth, rollback
+from cinq_collector_aws.resources import ELB
+
+
+class ELBList(BaseView):
+    URLS = ['/api/v1/elb']
+    MENU_ITEMS = [
+        MenuItem(
+            'browse',
+            'ELBs',
+            'elb.list',
+            'elb',
+            args={
+                'page': 1,
+                'count': 20,
+                'accounts': None,
+                'regions': None,
+                'num_instances': None
+            },
+            order=4
+        )
+    ]
+
+    @rollback
+    @check_auth(ROLE_USER)
+    def get(self):
+        self.reqparse.add_argument('page', type=int, default=1)
+        self.reqparse.add_argument('count', type=int, default=20, choices=[10, 20, 50])
```

Comment:
I think the counts here feel a bit low? Unless you have specific reasons I'd recommend sticking to the count : 100 in args and choices of 25, 50, 100 to match the other resources